core/cache/suite_test.go

```diff
@@ -0,0 +1,30 @@
+package cache_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/weaveworks/weave-gitops/pkg/testutils"
+)
+
+var k8sEnv *testutils.K8sTestEnv
+
+func TestMain(m *testing.M) {
+	os.Setenv("KUBEBUILDER_ASSETS", "../../tools/bin/envtest")
+
+	var err error
+	k8sEnv, err = testutils.StartK8sTestEnvironment([]string{
+		"../../manifests/crds",
```

Comment:
i don't think this path exists anymore?

Reply:
I spent close to zero effort on this, just copied over from `core/clustersmngr`

Reply:
perfect